resources/pgp-keys-map.list

```diff
@@ -428,8 +428,14 @@ org.apache.struts:struts-core:pom:1.3.8   = badSig
 org.apache.struts:struts-taglib:pom:1.3.8 = badSig
 org.apache.struts:struts-tiles:pom:1.3.8  = badSig
 
+org.apache.tomcat               = \
+                                  0x48F8E69F6390C9F25CFEDCD268248959359E722B, \
+                                  0x713DA88BE50911535FE716F5208B0AB1D63011C7, \
+                                  0xA9C5DF4D22E99998D9875A5110C01C5A2F6059E7
+
 org.apache.tomcat.embed         = \
                                   0x48F8E69F6390C9F25CFEDCD268248959359E722B, \
+                                  0x713DA88BE50911535FE716F5208B0AB1D63011C7, \
                                   0xA9C5DF4D22E99998D9875A5110C01C5A2F6059E7
```

Comment:
Maybe better and easier to maintain will be have one group with subgroup, like:
```
org.apache.tomcat.* = ....
```
we have the same keys